include/channel.h

```diff
@@ -113,12 +113,15 @@ struct Client;
 #define MODE_APASS	0x200000
 #define MODE_WASDELJOINS 0x400000 	/**< Not DELJOINS, but some joins
 					 * pending */
+
+#define MODE_MODERATENOREG 0x2000000    /**< +M Moderate unauthed users */
```

Comment:
Other channel modes are not surrounded by blank lines; this one should not be, either.

Reply:
FIxed